app/src/main/java/com/sdp13epfl2021/projmag/database/interfaces/UserdataDatabase.kt

```diff
@@ -152,4 +153,26 @@ interface UserdataDatabase {
         onSuccess: () -> Unit,
         onFailure: (Exception) -> Unit
     )
+
+    /**
+     * Uploads the user profile to a remote database
```

Comment:
Technically it's not 100% correct, because some implementation could be a partially local (offline) database.